contact.php

```diff
@@ -5,32 +5,32 @@
   $email = trim($_POST["email"]);
   $message = trim($_POST["message"]);
 
-
+  $error_message = array();
   if ($name == "" OR $email == "" OR $message == "") {
-    $error_message = "You must specify a value for name, email address and message.";
+    $error_message[] = "You must specify a value for name, email address and message.";
   }
 
   // Check for malicious form input
 
-  if (!isset($error_message)) {
+  if (!isset($error_message[0]) ){
     foreach ($_POST as $value) {
       if( stripos($value,'Content-Type:') != FALSE) {
-        $error_message = "There was a problem with the information you entered.";
+        $error_message[] = "There was a problem with the information you entered.";
       }
     }
   }
 
-  if (!isset($error_message) && $_POST["address"] != "") {
-    $error_message = "There was a problem with the form.";
+  if (!isset($error_message[0]) && $_POST["address"] != "") {
```

Comment:
It looks like you want errors to stack. You should stop checking anything about `$error_message` here because you want to unconditionally add the new error message to the array.